src/test/visual/visualComparison.spec.ts

```diff
@@ -26,6 +26,7 @@ test.describe('Visual Tests', () => {
 
       await expect(page).toHaveScreenshot(`${currentLanguage}_${screen.name}.png`, {
         fullPage: true,
+        maxDiffPixelRatio: 0.01,
```

Comment:
We shouldn't have difference at all between screenshots 
Please remove this line

Reply:
done:+1: